script/parse_options.jl

```diff
@@ -75,6 +75,39 @@ function get_agent_options(config::Dict{Any,Any}, ::HEMOptions{VIU}, solver::Sym
     )
 end
 
+function get_local_distribution_regulator_options(config::Dict{Any,Any})
+
+    rate_design, net_metering_policy, tou_suffix, planning_reserve_margin,
+    allowed_return_on_investment = parse(config, "Regulator", validators)
```

Comment:
We might want to have a different name. Maybe "LocalDistributionRateMaker"? (We probably should have named "Regulator" "RateMaker" to begin with ...)